src/context/LocationsContext.js

```diff
@@ -4,13 +4,19 @@ export const LocationsContext = createContext();
 
 const LocationsContextProvider = (props) => {
   const [locations, setLocations] = useState([]);
+  //state is updated in the Map component
+  const [recenter, setRecenter] = useState();
```

Comment:
There might be better variable names we could use here. Recenter sounds like something that happened more recently than something else 😅 

Maybe that could be `setNewCenter`?

What kinds of values are in `recenter`? Is it a boolean that means we need to recenter the map? Is it an object that contains a new lat/lng?

Reply:
newCenter & setNewCenter are much better names! newCenter is an object with new lat & lng. Should we start with an empty object?